content/en/tutorials/upgrading_eks.md

```diff
@@ -0,0 +1,81 @@
+---
```

Comment:
the tutorial section is becoming a bit crowded, I wonder if we should have this documentation living in `reference/aws/upgrading_eks.md` because this is only relevant to AWS

Reply:
I agree, but reference right now is for auto-generated docs at the moment based on the module yamls. We need to come up with a better overall breakdown/directory structure and I would love to hear any ideas you have about that.

Reply:
thank you